ding_availability.module

```diff
@@ -151,6 +151,7 @@ function ding_availability_items($provider_ids) {
  */
 function ding_availability_holdings($provider_ids) {
   $items = ding_provider_invoke('availability', 'holdings', $provider_ids);
+  $items = array_filter($items, function($v) { return $v['availability_information']; });
```

Comment:
Please add a comment explaining why this is necessary. The `availability_information` entry does not seem to be used further down the line.

Since the `array_filter` callback expects a boolean I would appreciate wrapping `$v['availability_information'];` in `empty()`. This should also prevent warnings if the entry is missing.